pkg/api/mlflow/service/artifact/storage/local_test.go

```diff
@@ -9,6 +9,60 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// TestGetArtifact for the local storage implementation.
```

Comment:
no need for comments here. for other tests we have no comments - which is fine, so lets have everything consistent.

Reply:
ok